tests/ballerina-unit-test/src/test/resources/test-src/expressions/lambda/arrow-expression.bal

```diff
@@ -175,3 +175,15 @@ function testArrowExprInObject() returns int {
     Bar f = new;
     return f.lambda(6);
 }
+
+string logString = "Global Text";
+
+function testArrowExprWithNoArguments() returns string {
+    function () returns string lambda = () => "Some Text " + logString;
+    return lambda();
+}
+
+function testArrowExprWithNoArgumentsAndStrTemplate() returns string {
+    function () returns string lambda = () => string`Some Text {{logString}}`;
+    return lambda();
+}
```

Comment:
Need test cases for closures as well with noArg arrow expressions.

Reply:
`logString` is a package level variable

Reply:
Nope, pkg level variables are anyway visible. Closures are outer scope from your current. So they need to have separate test cases.

Reply:
Ya that's true. Added a test case in 8cf15f5

Reply:
Ya that's true. Added a test case